### PR TITLE
testing/bpftrace: new aport

### PR DESCRIPTION
**Pull request by kaey**

### CODE REVIEW

File: testing/bpftrace/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Maintainer: Konstantin Kulikov <k.kulikov2@gmail.com>
+pkgname=bpftrace
+pkgver=0.9
+pkgrel=0
+pkgdesc="High-level tracing language for Linux eBPF"
+url="https://github.com/iovisor/bpftrace"
+arch="x86_64"
+license="Apache-2.0"
+makedepends="build-base cmake llvm7-dev llvm7-static clang-dev clang-static flex-dev bison elfutils-dev linux-headers bcc"
```
Comment (maxice8) [inline testing/bpftrace/APKBUILD]: Please cut this line at 80 columns

File: testing/bpftrace/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+source="$pkgname-$pkgver.tar.gz::https://github.com/iovisor/bpftrace/archive/v$pkgver.tar.gz"
+# Tests require root.
+options="!check"
```
Comment (maxice8) [inline testing/bpftrace/APKBUILD]: can't the tests be ran with the `checkroot` option ?

File: testing/bpftrace/APKBUILD (hunk 1 above)
Comment (maxice8) [reply testing/bpftrace/APKBUILD]: and build-base isn't required

Comment (kaey) [reply testing/bpftrace/APKBUILD]: done

File: testing/bpftrace/APKBUILD (hunk 2 above)
Comment (kaey) [reply testing/bpftrace/APKBUILD]: I've updated comment.
It's possible, but requires more work - download gmock, patch some tests and disable others (which depend on kernel config for ex).
I'll do all this, but a bit later.

Comment (maxice8) [reply testing/bpftrace/APKBUILD]: understandable

**Pull request closed without merging**